Author: Nilesh Patra <npatra974@gmail.com>
Description:
	* Initialize test so that test flags can be parsed during execution.
	* Change checksum to correct match - this has already been fixed upstream but new version has not been released yet.
Last-Changed: July 25, 2020

--- a/context_test.go
+++ b/context_test.go
@@ -11,6 +11,11 @@
 
 var save bool
 
+var _ = func() bool {
+	testing.Init()
+	return true
+}()
+
 func init() {
 	flag.BoolVar(&save, "save", false, "save PNG output for each test case")
 	flag.Parse()
@@ -191,7 +196,7 @@
 		dc.Pop()
 	}
 	saveImage(dc, "TestPushPop")
-	checkHash(t, dc, "e0d6724eb72033577811a55f7a642f6a")
+	checkHash(t, dc, "31e908ee1c2ea180da98fd5681a89d05")
 }
 
 func TestDrawStringWrapped(t *testing.T) {
